doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,149 +3,98 @@
 
-== Installation
-=== Manifest of dependencies for an unconnected deployment
+== Reference implementations
 
-{PRODUCT} now includes a manifest file with all of the Maven artifact dependencies required for operation, including development of services in {CENTRAL}. You can download the artifacts and provide them in a mirror repository in order to use {PRODUCT} without an outgoing connection to the public Internet.
+Three reference implementations are available that you can use as starter applications. They are provided in the *{PRODUCT} {PRODUCT_VERSION_LONG} Reference Implementations* download, available on the {PRODUCT} Software Downloads page in the Red Hat Customer Portal.
 
+.Employee Rostering reference implementation
+The employee rostering reference implementation enables you to create an application that assigns employees to shifts on various positions in an organization. For example, you can use the application to distribute shifts in a hospital between nurses, guard duty shifts across a number of locations, or shifts on an assembly line between workers.
 
-== {CENTRAL}
+.High available event-driven decisioning reference implementation
+A reference implementation for high available event-driven decisioning on {OPENSHIFT} is available. You can customize this reference implementation to deploy {PRODUCT_DROOLS} engine code that requires stateful processing, including rules developed with complex event processing, in an OpenShift environment. Doing this enables the {DECISION_ENGINE} to process complex event series with high availability.
 
-=== Git branch cloning
-{CENTRAL} now supports cloning specific branches from remote repositories.
+.Vehicle route planning reference implementation
+The vehicle route planning reference implementation enables you to create an application that solves a vehicle route planning problem with real-world maps, roads, and vehicles delivering goods to locations, each with a specific capacity. For more information, see the read me file in the vehicle routing ZIP file, included in the reference implementation download.
 
-=== Role-based access control
-{CENTRAL} now supports role-based access control for branches. Users can restrict access for a target branch to a specified collaborator. For example, if a user has security permission to update a project and has write permission for that branch based on the specified contributor type, the user can create new assets.
+== {CENTRAL}
 
-=== Test Scenarios
+=== Decision Model and Notation (DMN) designer improvements
 
-You can complete the following tasks in the test scenarios designer:
+The Decision Model and Notation (DMN) designer in {CENTRAL} includes the following notable enhancements:
 
-* Run a single test scenario.
-You can now run a single test scenario in the test scenarios table.
+* New *Documentation* tab that you can use to generate a report of your DMN model. You can print or download the report as an HTML file for offline use.
```

Comment:
this is really minor, but is "file for offline use" better than "file for use offline"? Just a little nit in my little brain.

Reply:
Well, TBH both sound fine to me. But this entry came from Stetson so I think I'll leave as it ;-)